libmscore/scorediff.h

```diff
@@ -36,7 +36,7 @@ enum class ItemType {
 //---------------------------------------------------------
 //   DiffType
 //---------------------------------------------------------
-
+#undef DELETE
```

Comment:
needed not just for MSVC, but also for MinGW, and as thirdparty/diff_match_oatch unconditionally `#undef`'s it , we're doing it here too

Reply:
It is not just me stumbling accross that, see https://musescore.org/en/node/277660